css/_site-navigation.scss

```
@@ -52,5 +52,10 @@
       color: var(--color-teal-400);
       font-weight: var(--semibold);
     }
+
+    m-icon {
+      position: relative;
+      top: var(--space-xxx-small);
```

```suggestion
      top: var(--space-xxx-small);
      margin-right: var(--space-xx-small);
```

This will space out the icon and anchor text a bit cleaner.